iModelCore/ECPresentation/Source/Shared/ECExpressions/ECExpressionContextsProvider.cpp

```diff
@@ -373,7 +373,10 @@ struct ECInstanceMethodSymbolsProvider : IECSymbolProvider
 
             if (BE_SQLITE_ROW == QueryExecutorHelper::Step(stmt))
                 {
-                evalResult.InitECValue() = ECValue(stmt.GetValueText(0));
+                LabelDefinitionPtr labelDefinition = LabelDefinition::FromString(stmt.GetValueText(0));
+                Utf8String expressionResult = labelDefinition->GetDisplayValue();

```

Comment:
`GetDisplayValue()` returns a reference - we should avoid assigning it to `Utf8String`, since that makes an unnecessary copy. I don't think we need the `expressionResult` variable at all.